Bottom: d20aa268d640b43da7c31e73cd8f49c448c0bc0a
Top:    f26dfc75799bcc469343ac76b609e69461c8d34f
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-09-30 17:04:47 -0700

Refresh of new-api-class-sbfile

---
diff --git a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
index 29b0320e062..fe75521e1b1 100644
--- a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
+++ b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
@@ -70,6 +70,7 @@ class FileHandleTestCase(lldbtest.TestBase):
             self.assertTrue(ret.Succeeded())
         return ret.GetOutput()
 
+
     @add_test_categories(['pyapi'])
     @no_debug_info_test
     def test_legacy_file_out_script(self):
